kms/apiv1/registry.go

```diff
@@ -13,12 +13,12 @@ type KeyManagerNewFunc func(ctx context.Context, opts Options) (KeyManager, erro
 
 // Register adds to the registry a method to create a KeyManager of type t.
 func Register(t Type, fn KeyManagerNewFunc) {
-	registry.Store(t, fn)
+	registry.Store(t.normalize(), fn)
 }
 
 // LoadKeyManagerNewFunc returns the function initialize a KayManager.
 func LoadKeyManagerNewFunc(t Type) (KeyManagerNewFunc, bool) {
-	v, ok := registry.Load(t)
+	v, ok := registry.Load(t.normalize())
```

Comment:
Have a slight preference for doing the `t.normalize()` inside of `.Load` and `.Store` instead of the method call here, but not a blocker.